tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "files": [
+        "index.js"
+      ],
+      "include": [
+        "lib/**/*.js",
+        "test/**/*.js"
+      ],
+      "compilerOptions": {
+        "target": "ES2020",
+        "lib": ["ES2020"],
+        "module": "commonjs",
```

Comment:
is `commonjs` (even as a compiler option) correct?
If these lines are copied from another place, this is a good time to check if some can be omitted

Reply:
it's a quick way to set `moduleResolution` to `node`, updated to use `module: ES2020` with an explicit `moduleResolution`